aspnetcore/tutorials/first-mvc-app/start-mvc.md

```diff
@@ -599,10 +599,12 @@ For Visual Studio for Mac, see the .NET 5 version of this tutorial.
 * In the **Create a new project** dialog, select **ASP.NET Core Web App (Model-View-Controller)** > **Next**.
 * In the **Configure your new project** dialog, enter `MvcMovie` for **Project name**. It's important to name the project *MvcMovie*. Capitalization needs to match each `namespace` when code is copied.
 * Select **Next**.
-* In the **Additional information** dialog, select **.NET 6.0 (Long-term support)**.
+* In the **Additional information** dialog:
+  * Select **.NET 7.0**.
+  * Uncheck **Do not use top-level statements**
```

Comment:
Shouldn't this be
Verify **Do not use top-level statements** is unchecked?
Isn't that the default?

Reply:
If they already selected it for a different project earlier, the default changes to selected, so it's safer to just say uncheck it.

Reply:
We'll see which version has more confusion, my RP verify vs your action verb.  Unlikely they'll have done it before, and if they have, verify takes care of it.